### PR TITLE
fixed "path too long" error on windows ci workflow target

### PR DESCRIPTION
**Pull request by adrtivv**
Resolves issue #10

Checked the [relay's ci workflow config](https://github.com/facebook/relay/blob/main/.github/workflows/ci.yml), didn't find a similar solution, they're probably using some other way to implement the same fix.

Cargo internally uses [libgit2](https://github.com/libgit2/libgit2) for git related workflows, and the library has [open issues](https://github.com/libgit2/libgit2/issues?q=longpaths+) regarding this that haven't been fixed yet. 

More info can be found here:- https://github.com/rust-lang/cargo/issues/13020

**TL;DR**:- Windows sucks, but made it work.

### CODE REVIEW

**Comment by rbalicki2** [comment]
Love it!! Thank you! I'll review and merge tomorrow. Thank you so much for doing this. This will be part of the 0.1.1 release!

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -32,10 +32,10 @@ jobs:
...
             build-name: isograph_cli
             artifact-name: isograph_cli-macos-arm64
-          # - target: x86_64-pc-windows-msvc
-          #   os: windows-latest
-          #   build-name: isograph_cli.exe
-          #   artifact-name: isograph_cli-bin-win-x64
+          - target: x86_64-pc-windows-msvc
+            os: windows-latest
+            build-name: isograph_cli.exe
+            artifact-name: isograph_cli-bin-win-x64
```
Comment (rbalicki2) [inline .github/workflows/ci.yml:38]: Could you please add a field here (e.g. longpaths: true), as well as the same field for the other OS's, and not have a special case for windows below?

**Review by rbalicki2 — CHANGES_REQUESTED** [review]
Love it!! Let's just use a separate field here (i.e. `longpaths: true`) instead of special casing it, and then let's get this landed.

Thank you so much for enabling Windows!!!

File: .github/workflows/ci.yml (hunk 1 above)
Comment (adrtivv) [reply .github/workflows/ci.yml:39]: added it similar to how it's done for the cargo **cross** compilation, it's not needed for UNIX based systems(linux, macos etc)

**Comment by rbalicki2** [comment]
🎉  Love it, thank you!!
